crucible-jvm/tool/Main.hs

```diff
@@ -129,6 +158,11 @@ cruxJVMConfig = Crux.Config
         $ Crux.ReqArg "TODO"
         $ \p opts ->
             Right $ opts { jarList = jarList opts ++ splitSearchPath p }
+      , Crux.Option ['b'] ["java-bin-dirs"]
+        "TODO"
```

Comment:
It would be nice to have a real description for this option.

Reply:
Fair enough. I copied over this `TODO` from the existing `TODO`s above, but it's easy enough to just fill them in. I've done so in a commit that I just pushed.